style.css

```diff
@@ -709,3 +709,282 @@ small {
     font-size: 14px;
     line-height: 20px;
 }
+.intro {
+    background-color: #373030;
+    position: relative;
+    padding: 40px 0 187px;
+    text-align: center;
+}
+.intro::after {
+    content: "";
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 80px;
+    z-index: 2;
+    background-color: #EA3829;
+}
+.intro__content {
+    max-width: 1310px;
+    padding: 0 15px;
+    margin: 0 auto;
+}
+.page__title {
+    margin: 0 0 20px;
+    font: 52px/62px var(--font-secondary);
+}
+.breadcrumbs {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    font-size: 14px;
+    line-height: 20px;
+    font-weight: 600;
+}
+.breadcrumbs li {
+    color: rgba(231, 238, 247, 0.66);
+    padding: 0 20px;
+    position: relative;
+}
+.breadcrumbs li::after {
+    content: "";
+    position: absolute;
+    left: 1px;
+    top: -2px;
+    height: 24px;
+    width: 2px;
+    border-radius: 2px;
+    background-color:rgba(208, 215, 251, 0.17);
+    z-index: 2;
+}
+.breadcrumbs li:first-child::after {
+    display: none;
+}
+.breadcrumbs a {
+    color: #F5F8FB;
+    text-decoration: none;
+    transition: opacity 0.2s linear;
+}
+.breadcrumbs a:hover {
+    opacity: 0.8;
+}
+.wrapper {
+    display: flex;
+    margin-inline: auto;
+    padding: 40px 15px 60px;
+    max-width: 1310px;
+}
+.sidebar {
+    flex-shrink: 0;
+    flex-grow: 0;
+    flex-basis: 328px;
+    margin-right: 36px;
+}
+.sidebar-box {
+    background-color: #1D1C28;
+    margin-bottom: 16px;
+    padding: 20px 15px;
+}
+.sidebar__title {
+    display: block;
+    font: 24px/32px var(--font-secondary);
+    margin-bottom: 16px;
+    position: relative;
+    padding-bottom: 16px;
+}
+.sidebar__title::after {
+    content: "";
+    position: absolute;
+    z-index: 2;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    height: 2px;
+    background-color: rgba(191, 197, 238, 0.12);
+}
+.sidebar__list,
+.tags {
+    list-style: none;
+    margin: 0;
+    padding: 0 10px;
```

Comment:
You can use logical property `padding-inline` if you need only horizontal paddings

Reply:
Done, thank you